creator-node/src/components/tracks/trackContentUploadManager.ts

```diff
@@ -302,6 +339,8 @@ async function batchSaveFilesToDisk({
           'segments',
           segmentFileName
         )
+
+        // @ts-ignore
```

Comment:
what are we ignoring here?

Reply:
That's the same AudiusLib stuff that you were dealing with in your PR